### DISCUSSION TITLE
Individual Object aka Klipper "EXCLUDE_OBJECT" errors

### DISCUSSION THREAD

**Discussion by tocguy** [supermerill/SuperSlicer#4040] [Q&A] [answered]
I am unexpectedly getting errors appearing in Octoprint's Klipper terminal in newly-sliced gcode files. They relate to the appearance of unknown "EXCLUDE_OBJECT" commands in the gcode file. I double-checked Superslicer's settings and the **_Complete individual objects_** checkbox is cleared. I toggled it ON and OFF again to no avail. The unwanted "EXCLUDE_OBJECT" commands remain. The gcode file contained only a single object anyway. With multi-object gcode files the problem is compounded. Is this a corruption in my slicer config? Is there something about Superslicer that I'm simply unaware about?

I re-sliced using the default slicer config and the issue did NOT repeat soooooo yeah, somehow Superslicer got a bee in its bonnet about  **_Complete individual objects_** .

Any idea why or how to discover what may be the cause?

**Comment by supermerill** [comment, marked as answer]
https://github.com/supermerill/SuperSlicer/issues/4049#issuecomment-1877293036
